README.md

```diff
@@ -139,17 +139,37 @@ Note some things:
 
 - The value for `HostName` is the hostname you got from Serverless.
 - The value for `Condition.KeyPrefixEquals` is whatever you've configured as `SCALED_FOLDER` in the environment variables.
-- The value for `HttpErrorCodeReturnedEquals` might be `403` or `404` based on your bucket and CloudFront settings.
+- The value for `HttpErrorCodeReturnedEquals` might be `403` or `404` based on your bucket. 403 when the objects in the bucket aren't public, 404 when they are.
 
 #### Let's see if it works!
 
 Upload an image to your bucket (make sure it's publicly readable), for example `foobar.jpg`.
 
-Now access this URL and see if it's worked: `https://<BUCKET_URL>/scaled/500x500/foobar.jpg.webp`.  
+Now access this URL and see if it's working: `https://<BUCKET_URL>/scaled/500x500/foobar.jpg.webp`.  
 If not, the first step in debugging is to go to the Lambda function in the AWS Console and check the CloudWatch logs.
 
 Good luck!
 
+### Use a CloudFront failover origin group
+
+In your AWS console, go to your CloudFront distribution. Under <strong>Origins</strong> to can add a second origin, beside your S3 bucket.
+
+Add an origin with the name "ImageScaler" and "Origin domain" (`0123456789.execute-api.eu-central-1.amazonaws.com`) should be the host of the API Gateway URL. Add the path of that url to "Origin path" (`default/resize?key=`).
```

Comment:
```suggestion
Add an origin with the name "ImageScaler" and "Origin domain" (`0123456789.execute-api.eu-central-1.amazonaws.com`) should be the host of the API Gateway URL. Add the path of that url to "Origin path" (`default/resize?key=scaled`).
```

Toch?

Reply:
Dit klopt niet. Bij de redirection rule moet dit wel, omdat het een replace is. Bij de origin is het een prepend. `abcdefg.cloudfront.net/scaled/123x123/foo.jpg` wordt `api-endpoint.com/default/resize?key=scaled/123x123/foo.jpg`. Het deel achter de hostname wordt erachter geplakt en daarmee bevraagd CloudFront de origin.

Reply:
Oh ja, precies, je hebt gelijk. 👍